pages/validators/preparing_for_genesis.md

```diff
@@ -67,35 +50,33 @@ cosmos_sdk_version: v0.47.3
 go: go version go1.19.9 <platform>
 name: dydxprotocol
 server_name: dydxprotocold
-version: 0.2.0-rc1
+version: <version>
 ```
 
-**Note:** We may release a newer binary for the actual network launch for `dydx-testnet-3`. Please stay tuned for updates in the Slack channels.
-
 ## Initialize your node
 
-Initialize the home directory for `dydxprotocold`. **Please note:** we are using `chain-id=dydx-testnet-3` for public testnet #3.
+Initialize the home directory for `dydxprotocold`. **Please note:** we are using `chain-id=dydx-mainnet-1` for public mainnet.
 
 ```bash
-# Use a new home directory for testnet 3.
-export HOME_TESTNET_3=~/.dydx-testnet-3
+# Use a new home directory for mainnet.
+export HOME_MAINNET_1=~/.dydx-mainnet-1
 # Choose a moniker as the custom username of your node. It should be human-readable.
 export DYDX_MONIKER=<moniker>
 # Configure chain ID.
-export CHAIN_ID=dydx-testnet-3
+export CHAIN_ID=dydx-mainnet-1
 # Init the home directory
-dydxprotocold init --chain-id=$CHAIN_ID --home=$HOME_TESTNET_3 $DYDX_MONIKER
+dydxprotocold init --chain-id=$CHAIN_ID --home=$HOME_MAINNET_1 $DYDX_MONIKER
 ```
 
 ## Saving/Recovering Consensus Keys
 
-Be sure to **make a copy** of the key pair json `priv_validator_key.json` and `node_key.json` under `$HOME_TESTNET_3/config`, as these key pairs are generated during `dydxprotocold init ....` and **cannot** be recovered later unless they were explicitly derived through the mnenomics.
+Be sure to **make a copy** of the key pair json `priv_validator_key.json` and `node_key.json` under `$HOME_MAINNET_1/config`, as these key pairs are generated during `dydxprotocold init ....` and **cannot** be recovered later unless they were explicitly derived through the mnenomics.
```

Comment:
"mnenomics": typo? or I don't know the word.

Reply:
it's a typo 🙏